entity-types/ext-triton/golden_metrics.yml

```diff
@@ -0,0 +1,48 @@
+nv_cpu_memory_total_bytes:
+  title: Provides the total CPU memory from the Triton inference server.
+  unit: BYTES
+  query:
+    select: latest(nv_cpu_memory_total_bytes)
+    from: Metric
+
+nv_cpu_memory_used_bytes:
```

Comment:
We use camel case names for the Golden metric names, one thing is the metric name in the origin `nv_cpu_memory_used_bytes` but another is the name you give it on NR side.
For example take a look here: https://github.com/newrelic/entity-definitions/blob/0dcd480cd8283a7c6a3b787abdcfb731d5cfce80/entity-types/ext-argocd/golden_metrics.yml#L8

Reply:
Hi @lujop,
As per suggestion, I have made changes.Kindly review it.